packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -451,7 +452,7 @@ export interface IrisGridState {
   columnHeaderGroups: readonly ColumnHeaderGroup[];
 }
 
-export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
+class IrisGrid extends Component<IrisGridProps, IrisGridState> {
```

Comment:
Why remove the named export?

Reply:
I'm pretty sure it wouldn't conflict w/ the index export since we explicitly list `IrisGrid` as an export, but I wanted to make sure it wouldn't conflict since we do `export * from './IrisGrid'`

Reply:
ok